ui/app/AppLayouts/Wallet/controls/NetworkFilter.qml

```diff
@@ -51,103 +46,85 @@ Item {
-            statusListItemTitle.font.weight: Font.Medium
-            statusListItemTitle.color: Theme.palette.baseColor1
-            title: root.multiSelection
-                        ? (root.enabledNetworks.count === root.allNetworks.count
-                            ? qsTr("All networks")
-                            : qsTr("%n network(s)", "", root.enabledNetworks.count))
-                        : d.selectedChainName
+    control.padding: 12
+    control.spacing: 0
+    control.rightPadding: 36
+    control.topPadding: 7
+    control.popup.width: 430
+
+    size: StatusComboBox.Size.Small
+
+    control.background: Rectangle {
+        height: 38
+        radius: 8
+        color: root.control.hovered ? Theme.palette.baseColor2 : "transparent"
+        border.color: Theme.palette.directColor7
+    }
+
+    contentItem: RowLayout {
+        spacing: 16
+        StatusSmartIdenticon {
+            Layout.alignment: Qt.AlignVCenter
             asset.height: 24
-            asset.width: asset.height
+            asset.width: 24
             asset.isImage: !root.multiSelection
             asset.name: !root.multiSelection ? Style.svg(d.selectedIconUrl) : ""
-            components:[
-                StatusIcon {
-                    width: 16
-                    height: 16
-                    icon: "chevron-down"
-                    color: Theme.palette.baseColor1
-                }
-            ]
-
-            onClicked: {
-                selectPopupLoader.active = !selectPopupLoader.active
-            }
+            active: !root.multiSelection
+            visible: active
         }
-    }
-
-    Row {
-        anchors.right: parent.right
-        anchors.bottom: parent.bottom
-        anchors.bottomMargin: Style.current.halfPadding
-        spacing: Style.current.smallPadding
-        visible: root.isChainVisible && chainRepeater.count > 0
-
-        Repeater {
-            id: chainRepeater
-            model: root.enabledNetworks
-            delegate: InformationTag {
-                tagPrimaryLabel.text: model.shortName
-                tagPrimaryLabel.color: model.chainColor
-                image.source: Style.svg("tiny/" + model.iconUrl)
+        StatusBaseText {
+            Layout.alignment: Qt.AlignVCenter
+            Layout.fillWidth: true
+            font.pixelSize: 13
+            font.weight: Font.Medium
+            elide: Text.ElideRight
+            lineHeight: 24
+            lineHeightMode: Text.FixedHeight
+            verticalAlignment: Text.AlignVCenter
+            text: root.multiSelection ? (d.allSelected ? qsTr("All networks") : "") : d.selectedChainName
+            color: Theme.palette.baseColor1
+            visible: !!text
+        }
+        Row {
+            spacing: -4
+            visible: !d.allSelected && chainRepeater.count > 0
+            Repeater {
+                id: chainRepeater
+                model: root.enabledNetworks
+                delegate: Image {
```

Comment:
Figma has border around second and any next item
![image](https://github.com/status-im/status-desktop/assets/11396062/fe918857-a76b-4d55-926e-21424321aa86)

In app there is no border
![image](https://github.com/status-im/status-desktop/assets/11396062/0f084aee-3ece-449e-a2a7-4a568d68eb3c)

Other than that really awesome job!

Reply:
<img width="148" alt="image" src="https://github.com/status-im/status-desktop/assets/60327365/7c78f732-68f3-4f3f-a08c-b771f279ffb6">
Hey thanks for pointing it out @Cuteivist. Fixed as you can see in screenshot above :)